ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1875,6 +1868,30 @@ AP_ServoRelayEvents *GCS_MAVLINK_Copter::get_servorelayevents() const
     return &copter.ServoRelayEvents;
 }
 
+#if ADVANCED_FAILSAFE == ENABLED
+AP_AdvancedFailsafe *GCS_MAVLINK_Copter::get_advanced_failsafe() const
+{
+    return &copter.g2.afs;
+}
+#endif
+
+MAV_RESULT GCS_MAVLINK_Copter::handle_flight_termination(const mavlink_command_long_t &packet) {
+    MAV_RESULT result = MAV_RESULT_FAILED;
+
+#if ADVANCED_FAILSAFE == ENABLED
+    if (super(packet) != MAV_RESULT_ACCEPTED) {
```

Comment:
@OXINARF Wait I think there is some confusion here. If AFS is enabled and says its capable of handling the termination then I allow it to manage everything. If AFS isn't enabled, and can't actually terminate the vehicle then I allow the existing behavior to terminate the copter. If that isn't whats happening then that is a bug. The goal is the vehicle should always be terminated.